spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -63,6 +64,21 @@ def get_unique_figname(dirname, root, ext):
             return osp.join(dirname, figname)
 
 
+@Slot()
+def copy_figure(fig, fmt):
```

Comment:
Please move this to be a method of `FigureBrowser`.

Reply:
The copy_figure function is used in ThumnailScrollBar Class and FigureCanvas Class.
If I convert this to method of FigureBrowser, the method become @staticmethod. Is it right?
If @slot decorator is the reason of fail, I can remove the decorator.

Reply:
I see. I will restore slot. And I will try rebase after studying rebase.
Would you like to change function to @staticmethod of FigureBrowser?